frontend/src/charts/DisparityBarChart.tsx

```diff
@@ -203,11 +234,26 @@ export function DisparityBarChart(props: DisparityBarChartProps) {
   const [ref, width] = useResponsiveWidth(
     100 /* default width during intialization */
   );
+
+  let data = props.data;
+  // Some standarized race names should be annotated with delimiters to indicate linebreaks
+  if (props.breakdownVar === RACE) {
+    data = props.data.map((data) => {
+      let copy = Object.assign({ ...data });
+      if (Object.keys(MULTILINE_RACE_MAP).includes(data.race_and_ethnicity)) {
+        copy["race_and_ethnicity"] =
+          MULTILINE_RACE_MAP[data.race_and_ethnicity];
+      }
+      return copy;
+    });
+  }
+  data.sort((a, b) => (a[props.breakdownVar] > b[props.breakdownVar] ? 1 : 0));
```

Comment:
You'd want 1, -1, and 0 instead of just 1 and 0. But you can just use the built-in `localeCompare` function for sorting which I think is better